lib/install/config/webpack/paths.yml

```diff
@@ -0,0 +1,7 @@
+# Restart webpack-watcher or webpack-dev-server if you make changes here
+paths:
+  src_path: app/javascript
+  config_path: config/webpack
+  node_modules_path: node_modules
+  dist_dir: packs
+  dist_path: public/packs
```

Comment:
@javan only issue is that that resolve_paths is plural...

I think being more verbose might be OK due to that.

When possible, we should be consistent with other Rails configs as well as the [webpack docs](https://webpack.js.org/concepts/output/#output-path). Webpack uses the path as an absolute one.

@gauravtiwari Since this is Yaml, let's add some docs on the different values and be clear that these are evaluated from the rails root.

This is looking GREAT!